core/controllers/cron.py

```diff
@@ -230,6 +230,25 @@ def get(self):
             logging.warning('Deletion jobs for JobModels entities kicked off.')
 
 
+class CronModelsCleanupHandler(base.BaseHandler):
+    """Handler for cleaning up models that are marked as deleted and marking
+    specific types of models as deleted.
+    """
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        """Cron handler that is run every week and hard-deletes all models that
+        are marked as deleted (have deleted field set to True) and were
+        last updated more than some period of time. Also, for some types of
```

Comment:
```suggestion
        """Cron handler to hard-delete all models that are marked as deleted X weeks ago. Also, for some types of ....
```

Should we change it to this?

Reply:
Note: `that is run every week` this info is not required in this file/handler. (As it's defined in cron.yml context and can be changed in future.)

Reply:
Yeah makes sense, done.